src/ape/managers/chain.py

```diff
@@ -1075,3 +1092,12 @@ def set_balance(self, account: Union[BaseAddress, AddressType], amount: Union[in
             amount = int(amount, 16)
 
         return self.provider.set_balance(account, amount)
+
+    def get_receipt(self, transaction_hash: str) -> ReceiptAPI:
```

Comment:
Note that this was originally intended to be here in the original ticket! (was called `get_transaction`) then:
https://github.com/ApeWorX/ape/issues/129

Just thought I'd add that, it feels validating